Automattic/jetpack
# Move Backup UI from plugin to package

Replaces and closes #23399




#### Changes proposed in this Pull Request:

Moves all the UI of the Backup plugin into the Backup package to enable us to easily bundle it inside Jetpack plugin.

#### Jetpack product discussion


p1HpG7-dFB-p2

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Test all aspects of Backup plugin and it should work the same way as previous
* Make sure i18n strings are using the proper domain name

Checkout  https://github.com/Automattic/jetpack/pull/23532

* Test it alongside Jetpack plugin
* Test it only with Jetpack plugin active
